lib/superstore/types/json_type.rb

```diff
@@ -1,26 +1,6 @@
 module Superstore
   module Types
     class JsonType < BaseType
-      OJ_OPTIONS = {mode: :compat}
-      def encode(data)
-        data
-      end
-
-      def decode(str)
-        str
-      end
-
-      def typecast(data)
-        if data.acts_like?(:time) || data.acts_like?(:date)
-          data.as_json
-        elsif data.is_a?(Array)
-          data.map { |d| typecast(d) }
-        elsif data.is_a?(Hash)
-          data.each_with_object({}) { |(key, value), hash| hash[key] = typecast(value) }
-        else
-          data
-        end
-      end
```

Comment:
Made object instantiation very slow, especially ones that used the json data type extensively.